blocks/newslist/newslist.js

```diff
@@ -377,21 +378,25 @@ export default async function decorate(block) {
     searchHeader.classList.add('search-header-container');
     const form = `
       <form action="/search" method="get" id="search-form">
-        <input type="text" id="search-input" title="${pKeywords}" placeholder="${pKeywords}" name="q" value="${query}" size="40" maxlength="60">
+        <input type="text" id="search-input" title="${pKeywords}" placeholder="${pKeywords}" name="q" value="" size="40" maxlength="60">
         <input type="submit" title="${pSearch}" value="${pSearch}">
       </form>
     `;
     if (query) {
       searchHeader.innerHTML = `
       ${form}
-      <h2>Results for "${query}"</h2>
+      <h2></h2>
       <div class="search-sub-header">
         <h3> ${shortIndex.length > 0 ? 'ALL RESULTS' : '0 RESULTS WERE FOUND'} </h3>
         <div class="search-sub-header-right">
           ${`Showing ${start + 1} - ${Math.min(end, shortIndex.length)} results`}
         </div>
       </div>
       `;
+      const heading = searchHeader.querySelector('h2');
```

Comment:
@raymond-h-dev any specific reason to explicitly set using js? Isn't this same as adding this in the 
```
<h2>Results for "${query}"</h2>
```
?

Reply:
Thank you @raymond-h-dev for the explanation.
But by the time you're putting the `query` in `h2`, its already sanitized right? so, there shouldn't be any problem using, right?

```
<h2>Results for "${query}"</h2>
```

Reply:
yes, that already sanitized and clean up the special characters but not the other way around. for example if you type string `&lt;/TITLE&gt;&lt;SCRIPT&gt;alert(\"XSS\");&lt;/SCRIPT&gt; ` in the home search box and submit it, it will convert as script tag(see images below) that might execute some threats to our page. 

![image](https://github.com/hlxsites/accenture-newsroom/assets/44294653/3a29f1d0-789d-4b53-bc37-ad380705a78b)